lib/aweplug/books/google_books.rb

```diff
@@ -48,15 +48,15 @@ def initialize site, push_to_searchisko
         @client = google_client(site, authenticate: site.authenticate_google_books_api)
         @books = @client.discovered_api(BOOKS_API_SERVICE_NAME, BOOKS_API_VERSION)
         @searchisko = Aweplug::Helpers::Searchisko.default site, 360
+        @env_param = {
+            :q => "isbn:#{data['isbn']}"}
       end
 
       def get data
+        @env_param.merge!(:country => ENV['country_code']) if ENV['country_code']
```

Comment:
Will fix and resubmit.

~Dan
On Feb 6, 2015, at 11:16 AM, Jason Porter <notifications@github.com> wrote:

> In lib/aweplug/books/google_books.rb:
> 
> >        end
> >  
> >        def get data
> > +        @env_param.merge!(:country => ENV['country_code']) if ENV['country_code']
> Uppercase country_code
> 
> —
> Reply to this email directly or view it on GitHub.
>